elixir/apps/domain/lib/domain/auth/adapters/microsoft_entra/jobs.ex

```diff
@@ -0,0 +1,201 @@
+defmodule Domain.Auth.Adapters.MicrosoftEntra.Jobs do
+  use Domain.Jobs.Recurrent, otp_app: :domain
+  alias Domain.{Auth, Actors}
+  alias Domain.Auth.Adapters.MicrosoftEntra
+  require Logger
+
+  every minutes(5), :refresh_access_tokens do
```

Comment:
I think it would be a good idea to be able to control this timing through some type of config.  That said, if we put something in terraform, isn't that going to force a redeploy due to how our deployment works?

Ideally I could see this being set by some type of Admin Ops dashboard in the future, but that may be a ways off.

Reply:
Brian is right, redeploying and changing one line with a typical workflow is the same "time to deliver", only difference is that somebody can go manually do `terraform apply` to update it in case of emergencies, which, I think, not worth it now as there are other ways to do it with Elixir.

Reply:
Yeah forgot about the apply step, was thinking it was just an env read on boot